modules/claims_api/app/services/claims_api/disability_compensation/docker_container_service.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+require 'pdf_generator_service/pdf_client'
+require 'claims_api/v2/disability_compensation_evss_mapper'
+require 'evss_service/base'
+
+module ClaimsApi
+  module DisabilityCompensation
+    class DockerContainerService < ClaimsApi::Service
+      def upload(claim_id)
+        log_service_progress(claim_id, 'docker_service',
+                             'Docker container service started')
+
+        auto_claim = get_claim(claim_id)
+        # Reset for a rerun on this
+        set_pending_state_on_claim(auto_claim) unless auto_claim.status == pending_state_value
+
+        evss_data = get_evss_data(auto_claim)
+
+        log_service_progress(claim_id, 'docker_service',
+                             'Submitting mapped data to Docker container')
+
+        evss_res = evss_service.submit(auto_claim, evss_data)
+
+        log_service_progress(claim_id, 'docker_service',
+                             "Successfully submitted to Docker container with response: #{evss_res}")
+        # update with the evss_id returned
+        auto_claim.update!(evss_id: evss_res[:claimId])
+        # clear out the evss_response value on successful submssion to docker container
+        clear_evss_response_for_claim(auto_claim)
```

Comment:
since this is synchronous and we won't be retrying, a failure would create a new instance in the DB, so anything that makes it here would not have an EVSS Response - this could be a tech debt / cleanup as well.